src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/telemetry/ProbationBookingCreatedTelemetryEvent.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.telemetry
+
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.common.toIsoDateTime
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.VideoBooking
+import java.time.temporal.ChronoUnit
+
+class ProbationBookingCreatedTelemetryEvent(private val booking: VideoBooking) :
+  MetricTelemetryEvent("BVLS-probation-booking-created") {
+
+  init {
+    require(booking.isProbationBooking()) {
+      "Cannot create probation created metric, video booking with ID '${booking.videoBookingId}' is not a probation booking."
+    }
+  }
+
+  override fun properties() =
+    mapOf(
+      "video_booking_id" to "${booking.videoBookingId}",
+      "team_code" to booking.probationTeam!!.code,
```

Comment:
Ticket had probation_team_code and probation_meeting_type as keys.  Happy to change to that.